paddle/fluid/operators/mkldnn/layer_norm_mkldnn_op.cc

```diff
@@ -133,20 +132,7 @@ class LayerNormMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
 
     std::shared_ptr<mkldnn::memory> scaleshift_memory;
```

Comment:
Can you put this `std::shared_ptr<mkldnn::memory> scaleshift_memory;` inside `if (with_scaleshit)` ?

Reply:
Ok